### ISSUE TITLE
API: Some fields come in floating point

### ISSUE THREAD

**Issue by markusrubey** [where2help/where2help#171]
I had a look at the `/events` endpoint and noticed that a couple fields come in floating point. Is this on purpose? An `id` from the `/user` endpoint on the other hand returns integers, as expected. 
```
  {
    "id": 4.0,
    "title": "Was Lives On Pluto",
    "description": "Fashion axe cray vinegar. Biodiesel fashion axe roof.",
    "address": "1140, Linzer Strasse 42",
    "lat": 48.193442201683,
    "lng": 16.3111526090372,
    "state": "published",
    "created_at": "2016-06-20T23:01:10.996+02:00",
    "updated_at": "2016-06-20T23:01:13.062+02:00",
    "organization_name": "Schulz, Amberg und Graf",
    "url": "http://staging-where2help.herokuapp.com/api/v1/events/4?locale\u003dde",
    "shifts": [
      {
        "id": 11.0,
        "event_id": 4.0,
        "starts_at": "2016-06-23T10:35:22.991+02:00",
        "ends_at": "2016-06-23T12:35:22.991+02:00",
        "volunteers_needed": 8.0,
        "volunteers_count": 0.0,
        "created_at": "2016-06-20T23:01:10.998+02:00",
        "updated_at": "2016-06-20T23:01:10.998+02:00",
        "current_user_assigned": false
      },
  .
  .
  .
```

**Comment by klappradla** [comment]
> Biodiesel fashion axe roof ❤️💛💚💙💜

Besides the ID part being strange, I also don't really see a point in volunteer counts being floats.

**Comment by haslinger** [comment]
No, not on purpose. Maybe the switch to Rails 5. Will look into it.

**Comment by markusrubey** [comment]
Strange indeed. And faulty request return Http Status Codes as float. :see_no_evil:
```
{"status":404.0,"error":"Not Found"}
```

**Comment by haslinger** [comment]
I cannot confirm that problem, just tested /events and got back (on staging):

`[{"id":4,"title":"Was Lives On Pluto","description":"Fashion axe cray vinegar. Occupy scenester actually mlkshk leggings photo booth craft beer squid. Swag banh mi asymmetrical bushwick. Biodiesel fashion axe roof.","address":"1140, Linzer Strasse 42","lat":48.193442201683,"lng":16.3111526090372,"state":"published","created_at":"2016-06-20T23:01:10.996+02:00","updated_at":"2016-06-20T23:01:13.062+02:00","organization_name":"Schulz, Amberg und Graf","url":"http://staging-where2help.herokuapp.com/api/v1/events/4?locale=de","shifts":[{"id":11,"event_id":4,"starts_at":"2016-06-23T10:35:22.991+02:00","ends_at":"2016-06-23T12:35:22.991+02:00","volunteers_needed":8,"volunteers_count":0,"created_at":"2016-06-20T23:01:10.998+02:00","updated_at":"2016-06-20T23:01:10.998+02:00","current_user_assigned":false},{"id":12,"event_id":4,"starts_at":"2016-06-23T12:35:22.991+02:00","ends_at":"2016-06-23T14:35:22.991+02:00","volunteers_needed":12,"volunteers_count":0,"created_at":"2016-06-20T23:01:10.999+02:00","updated_at":"2016-06-20T23:01:10.999+02:00","current_user_assigned":false}]},{"id":1,"title":"Gehen Und Laufen","description":"Beard mixtape schlitz leggings mustache. Cardigan truffaut pabst green juice bespoke pour-over before they sold out vinegar. Slow-carb typewriter tousled plaid pickled shoreditch skateboard migas." ...`

So no floats in here ...

**Comment by markusrubey** [comment]
That's weird. Thanks for testing @haslinger!
